setup.py

```diff
@@ -0,0 +1,41 @@
+import os
+from setuptools import setup, find_packages
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+def content(path, cwd=None):
+    if cwd:
+        fpath = os.path.join(cwd, path)
+    else:
+        fpath = path
+    with open(fpath) as fd:
+        return fd.read()
+
+
+setup(
+    name="fasjson-client",
+    author="Fedora Infrastructure Team",
+    version="0.0.1",
+    author_email="infrastructure@lists.fedoraproject.org",
+    description="Fedora Account System OpenAPI client",
+    long_description=content("README.md", cwd=here),
+    url="https://github.com/fedora-infra/fasjson-client",
+    packages=find_packages(exclude=("test,")),
+    include_package_data=True,
+    # Possible options are at https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        "Development Status :: 2 - Pre-Alpha",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Intended Audience :: Developers",
+        "Topic :: Security",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    license="LGPLv3",
+    install_requires=content('requirements.txt').split('\n'),
+    tests_requires=content('test_requirements.txt').split('\n'),
```

Comment:
Are you sure this works, with the empty lines and the comments in that file? I think they should be filtered out by the `content` function.